config/config.yaml

```diff
@@ -39,3 +39,5 @@ tunings:
     iterations: 5                                        # Iterations to loop before stopping the watch, it will be replaced with infinity when the daemon mode is enabled
     sleep_time: 60                                       # Sleep duration between each iteration
     daemon_mode: True                                    # Iterations are set to infinity which means that the cerberus will monitor the resources forever
+    database_path: /tmp/cerberus.db                      # Path where cerberus database needs to be stored
```

Comment:
Maybe move the path and resue options under a new config option called database instead of tunings?